### PR TITLE
enhancement : added new boilerplate

### PR DESCRIPTION
**Pull request by mogiiee**
i have added a new entry

### CODE REVIEW

**Comment by mogiiee** [comment]
i have resolved the merge conflict

File: submodules.json [hunk 1]
```diff
@@ -1,17 +1,28 @@
...
-        "name" : "node-boilerplate",
-        "url" : "https://github.com/hagopj13/node-express-boilerplate",
-        "tags" : ["node", "nodejs"]
-    }
-    {
-        "name": "repo-boilerplate",
-        "url": "https://github.com/mintel/repo-boilerplate",
-        "tags" : ["none"]
-    }
+
```
Comment (mintbomb27) [inline submodules.json:2]: You might've removed a `{` while resolving the merge conflict. Please do keep that.

File: submodules.json [hunk 2]
```diff
@@ -1,17 +1,28 @@
...
+  },
+  {
+    "name": "android-boilerplate",
+    "url": "https://github.com/ribot/android-boilerplate",
+    "tags": ["android", "mvvm", "rxjava"]
+  },
+  {
+    "name": "repo-boilerplate",
+    "url": "https://github.com/mintel/repo-boilerplate",
+    "tags" : ["none"]
```
Comment (mintbomb27) [inline submodules.json:25]: If you have no tags, the repo won't be displayed in the results. You could add tags like `basic`,`repo` or stuff like that.

**Review by mintbomb27 — CHANGES_REQUESTED** [review]
Please check out the changes requested. Thank you for your contribution! :)

**Comment by mogiiee** [comment]
@mintbomb27 please have a look at it now.

**Review by mintbomb27 — COMMENTED** [review]
Hey! Would be great if you could find some other repo that contains boilerplate for some framework's or language's code instead of a basic repo boilerplate. Can keep this PR open for your contribution. Marking accepted.

**Comment by mogiiee** [comment]
@mintbomb27 could you please check it now... i have added a boilerplate for fastapi with mongodb in the correct order.

**Pull request closed without merging**